app/controllers/application_controller.rb

```diff
@@ -157,7 +157,16 @@ def record_exists e
 
   def forbidden e
     logger.debug "#{e}\n#{e.backtrace.join "\n"}"
-    head :forbidden
+    if e.message != "ApplicationController::Forbidden"
+      render json: {
```

Comment:
I think this should render text unless JSON is explicitly requested in an `Accept` header.